scalanet/src/io/iohk/scalanet/peergroup/UDPPeerGroup.scala

```diff
@@ -64,25 +62,24 @@ class UDPPeerGroup[M](val config: Config)(
 
   private def removeChannel(channel: ChannelImpl): Unit = {
     activeChannels.remove(channel.channelId)
-    channel.messageSubject.onComplete()
```

Comment:
Is there another way we can signal that the channel is closed? I was looking for a `ChannelEvent` representing the closing of the socket, and so far this was the closest I found.

Reply:
yes, that was also my first thouht, adding new channel event to signal finishg.